services/tvoc.md

```diff
@@ -0,0 +1,25 @@
+# Total volatile organic compound
+
+Measures equivalent TVOC levels.
+
+    identifier: 0x12a5b597
+    extends: _sensor
+    camel: TVOC
+
+## Registers
+
+     ro TVOC: u16 ppm { typical_min=0, typical_max=1187 }  @ reading
+
+Total volatile organic compound readings.
+
+    const min_TVOC?: u16 ppm @ min_reading
+
+Minimum measurable value
+
+    const max_TVOC?: u16 ppm @ max_reading
+
+Minimum measurable value
+
+    const conditioning_period?: u16 min @ 0x180
```

Comment:
how about u16.16 s ? I would rather not add another unit...

Reply:
u32 s